chains/evm/cli/cli.go

```diff
@@ -52,6 +61,62 @@ func BindEVMCLIFlags(evmRootCLI *cobra.Command) {
 
 }
 
+func writeCliIODataToFile(cmd *cobra.Command) {
```

Comment:
I do not see that this function been used in all CLIs, means now its just sits here and after merge no logging will be happen, or i am missing something?